src/logger/fileWriter.c

```diff
@@ -34,47 +35,55 @@
 #define WRITE_SUCCESS  0
 #define WRITE_FAIL     EOF
 
-typedef struct _FileBuffer {
-    char buffer[FILE_BUFFER_SIZE];
-    size_t index;
-} FileBuffer;
-
 static FIL *g_logfile;
 static xQueueHandle g_LoggerMessage_queue;
-static FileBuffer fileBuffer = {"", 0};
-
+static struct ring_buff file_buff;
 
-static void clear_file_buffer() {
-        fileBuffer.index = 0;
-        fileBuffer.buffer[0] = '\0';
+static void error_led(bool on)
+{
+        on ? LED_enable(3) : LED_disable(3);
 }
 
-static int writeFileBuffer()
+static FRESULT flush_file_buffer()
 {
-        unsigned int bw;
-        const int rc = f_write(g_logfile, fileBuffer.buffer,
-                               fileBuffer.index, &bw);
-        clear_file_buffer();
-        return rc;
+        FRESULT res = FR_OK;
+        char tmp[32];
+
+        pr_info("Flushing file buffer\r\n");
```

Comment:
everywhere we do logging we should prefix it with the module / area doing the logging.  format is <moduleName>: <msg>

So in this case:

pr_info("fileWriter: flushing file buffer\r\n");

applies throughout.

Also, this particular message should be debug.